### PR TITLE
Remove clang-extract support

### PR DESCRIPTION
**Pull request by marcosps**

### CODE REVIEW

File: klp-build.1 [hunk 1]
```diff
@@ -120,33 +120,9 @@ Apply patches found by
...
-.TP
 .BI --codestreams " CODESTREAMS CODESTREAMS"
 Apply diff on two different codestreams.
 .TP
-.BI --type " {ccp,ce}"
-Choose between
-.BR klp-ccp (1)
-and
-.BR clang-extract "(1)."
-.RE
```
[the remark this answers is not in the document]
Comment (fgyanz) [reply klp-build.1:148]: Something like:
```
.RE
.TP
.B format-patches
```

**Comment by fgyanz** [comment]
Error when running `extract` (without `ignore-errors`):
```
File "/klpbuild/cmd.py", line 191, in main_func
    args.avoid_ext, args.ignore_errors).run()
                    ^^^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'ignore_errors'
```

**Review by fgyanz — CHANGES_REQUESTED** [review]

**Comment by marcosps** [comment]
> Error when running `extract` (without `ignore-errors`):
> 
> ```
> File "/klpbuild/cmd.py", line 191, in main_func
>     args.avoid_ext, args.ignore_errors).run()
>                     ^^^^^^^^^^^^^^^^^^
> AttributeError: 'Namespace' object has no attribute 'ignore_errors'
> ```

Ouch, that's true! I forgot to rerun the tests maybe...

**Comment by fgyanz** [comment]
I'm unsure about where exactly, but apparently for each `extract` thread we are instantiating the configuration file.
```
 Generating livepatches for 11 file(s) using 16 workers...
		Codestream	File
         (1/11) 15.6rtu0        fs/smb/client/cifs_debug.c
         (2/11) 15.6rtu1        fs/smb/client/cifs_debug.c
         (3/11) 15.6rtu2        fs/smb/client/cifs_debug.c
         (4/11) 15.6rtu3        fs/smb/client/cifs_debug.c
         (5/11) 15.6rtu4        fs/smb/client/cifs_debug.c
         (6/11) 15.6u0          fs/smb/client/cifs_debug.c
         (7/11) 15.6u1          fs/smb/client/cifs_debug.c
         (8/11) 15.6u2          fs/smb/client/cifs_debug.c
         (9/11) 15.6u3          fs/smb/client/cifs_debug.c
        (10/11) 15.6u4          fs/smb/client/cifs_debug.c
        (11/11) 15.6u5          fs/smb/client/cifs_debug.c
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
Loading user configuration from '/home/fgonzalez/.config/klp-build/config'
```
Could this be optimized so that we only do it once?

**Comment by marcosps** [comment]
The issue seems to be that right now I'm instantiating the CCP class inside the process function, and CCP also inherits from Config. One think that I was planning to do was to make Extract be the CCP class, now that we don't have to support more than one backend anymore.

For now I'll try to now depend on Config class anymore. I'll push a new update to this PR later today. Thanks for checking this out!

**Comment by marcosps** [comment]
@fgyanz should be fixed now!

**Comment by fgyanz** [comment]
LGTM! Good job!

**Pull request merged**